src/terraform/devcontainer-feature.json

```diff
@@ -41,12 +41,10 @@
                 "ms-azuretools.vscode-azureterraform"
             ],
             "settings": {
-                "terraform.languageServer": {
-                    "enabled": true,
-                    "args": [
-                        "serve"
-                    ]
-                },
+                "terraform.languageServer.enable": true,
```

Comment:
Can we bump the patch version here to `1.0.5`? We follow semantic versioning. https://github.com/devcontainers/features/blob/main/src/terraform/devcontainer-feature.json#L3

Features are released every time a PR is merged, these changes won't take effect unless the version specified here is different than the ones already published 😄

Reply:
Done!